src/messages/MMonCommand.h

```diff
@@ -41,10 +42,26 @@ class MMonCommand final : public PaxosServiceMessage {
 public:
   std::string_view get_type_name() const override { return "mon_command"; }
   void print(std::ostream& o) const override {
+    cmdmap_t cmdmap;
```

Comment:
please move the implementation into .cc.

Reply:
there is currently no .cc file in https://github.com/ceph/ceph/tree/master/src/messages and for the simplicity of backport until nautilus, can we agree to leave it here?

Reply:
fair enough. will create a follow-up PR to address this.